File: ssh-frontend/src/services/commandService.js
import axios from "axios";


export default class commandService {
    getAllCommandByCredentialIdAndPageable(id,pageNo,pageSize){
        return axios.get("http://localhost:8080/api/command/getCommandByCredentialId?credentialId="+id+"&pageNo="+pageNo+"&pageSize="+pageSize)
    }
}

File: ssh-backend/src/main/java/com/ssh/service/abstracts/CommandService.java
package com.ssh.service.abstracts;

import com.ssh.core.utilities.bean.Command;
import com.ssh.core.utilities.results.DataResult;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
public interface CommandService {
    DataResult<Command> setCommand(Command command);

    DataResult<List<Command>> getCommands(int credentialId, int pageNo, int pageSize);
}
